### PR TITLE
Update Outbox because fan mail's been deprecated

### PR DESCRIPTION
**Pull request by radialarch**
Outbox was still checking if the send fan mail button had loaded on people's blogs every 200 ms, even though the button's gone. Technically the page for fan mail still _exists _ at tumblr.com/send, but I don't think you can actually use it to send new fan mail. 

So this is just outbox.js with all the code referencing fan mail removed. The saving asks functionality still looks fine (I sent myself an ask to check). Should mention I'm not sure what happens to fan mail that's still saved in someone's outbox? The 50-message limit means I don't have any fan mail left in my outbox to check.

console log on a blog

### CODE REVIEW

**Comment by nightpool** [comment]
looks like there are go-to-dash changes in here too?

File: Extensions/outbox.js [hunk 1]
```diff
@@ -530,63 +436,8 @@ XKit.extensions.outbox = new Object({
 
 	},
 
-	render_fan_mail: function(obj, m_id) {
```
Comment (nightpool) [inline Extensions/outbox.js]: You can probably keep all the render stuff in. Otherwise, looks good to me.

**Comment by radialarch** [comment]
Oh, yeah, I was gonna submit a pull request for #1014 but then #1024 beat me to it? I think we did it pretty similarly, at any rate. Should I...revert the go-to-dash changes and let you merge just the outbox commit? (Welp, I'm super new to github, idk what I'm doing D:)

I put back the render fanmail, check that over!

**Comment by nightpool** [comment]
Probably a cherry-pick or an interactive rebase is what you're going to want to do here.

**Comment by nightpool** [comment]
Okay, I'm going to open another PR here so I can clean up the commit history. Sound good?

**Comment by nightpool** [comment]
Re-Opened at #1027

**Comment by nightpool** [comment]
@radialarch Okay, this got merged as #1027. The PR was opened by me, but the commit is credited to you. Thanks for the commit!

**Comment by nightpool** [comment]
FYI it looks like the person who did #1024 isn't active enough to respond to my comments, so if you can integrate the changes I mentioned in my review and PR it, I can approve that too.

Thanks! feel free to drop by our [gitter](https://gitter.im/new-xkit/XKit) whenever to hang out too!

**Pull request closed without merging**